cts/cts-exec.in

```diff
@@ -988,28 +988,28 @@ class TestOptions(object):
                 self.options['run-only-pattern'] = args[i+1]
                 skip = True
 
-def show_usage():
-    """ Show command usage """
-
-    print("usage: " + sys.argv[0] + " [options]")
-    print("If no options are provided, all tests will run")
-    print("Options:")
-    print("\t [--help | -h]                        Show usage")
-    print("\t [--list-tests | -l]                  Print out all registered tests.")
-    print("\t [--run-only | -r 'testname']         Run a specific test")
-    print("\t [--verbose | -V]                     Verbose output")
-    print("\t [--timeout | -t 'floating point number']"
-            "\n\t\tUp to how many seconds each test case waits for the daemon to be initialized."
-            "\n\t\tDefaults to 2. The value 0 means no limit.")
-    print("\t [--force-wait | -w]"
-            "\n\t\tEach test case waits the default/specified --timeout for the daemon without tracking the log.")
-    if BuildOptions.REMOTE_ENABLED:
-        print("\t [--pacemaker-remote | -R             Test pacemaker-remoted binary instead of pacemaker-execd")
-    print("\t [--run-only-pattern | -p 'string']   Run only tests containing the string value")
-    print("\n\tExample: Run only the test 'start_stop'")
-    print("\t\t " + sys.argv[0] + " --run-only start_stop")
-    print("\n\tExample: Run only the tests with the string 'systemd' present in them")
-    print("\t\t " + sys.argv[0] + " --run-only-pattern systemd")
+    def show_usage(self):
+        """ Show command usage """
+
+        print("usage: " + sys.argv[0] + " [options]")
+        print("If no options are provided, all tests will run")
+        print("Options:")
+        print("\t [--help | -h]                        Show usage")
+        print("\t [--list-tests | -l]                  Print out all registered tests.")
+        print("\t [--run-only | -r 'testname']         Run a specific test")
+        print("\t [--verbose | -V]                     Verbose output")
+        print("\t [--timeout | -t 'floating point number']"
+                "\n\t\tUp to how many seconds each test case waits for the daemon to be initialized."
+                "\n\t\tDefaults to 2. The value 0 means no limit.")
+        print("\t [--force-wait | -w]"
+                "\n\t\tEach test case waits the default/specified --timeout for the daemon without tracking the log.")
+        if BuildOptions.REMOTE_ENABLED:
+            print("\t [--pacemaker-remote | -R             Test pacemaker-remoted binary instead of pacemaker-execd")
+        print("\t [--run-only-pattern | -p 'string']   Run only tests containing the string value")
+        print("\n\tExample: Run only the test 'start_stop'")
+        print("\t\t " + sys.argv[0] + " --run-only start_stop")
+        print("\n\tExample: Run only the tests with the string 'systemd' present in them")
+        print("\t\t " + sys.argv[0] + " --run-only-pattern systemd")
```

Comment:
Not for this PR, but eventually I'd like to move to the argparse module for all Python executables, which will get rid of a lot of custom arg parsing and allow us to share some argument definitions across executables